src/calendar-list/style.ts

```diff
@@ -30,7 +30,8 @@ export default function getStyle(theme: Theme = {}) {
       left: 0,
       right: 0,
       top: 0,
-      backgroundColor: appStyle.calendarBackground
+      backgroundColor: appStyle.calendarBackground,
+      paddingHorizontal: 15
```

Comment:
This extra padding breaks the layout of the timeline calendar - try to uncomment the `numberOfDays={3}` in the `timelineCalendarScreen.tsx` and see the padding of the days' names

Reply:
Also, the days' names don't change when moving to the next 3 days

Reply:
Header alignment fixed

Reply:
There is an issue with the dates but I do see the day names are changing...